### ISSUE TITLE
Is live server reloading twice?

### ISSUE THREAD

**Issue by camilopedroso** [ritwickdey/vscode-live-server#175]
## I'm submitting a...
![loading_twice](https://user-images.githubusercontent.com/872336/43359962-709d04fc-9283-11e8-827d-a1030233159b.png)

[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[ ] Documentation issue or request
[x] Other: help request
```

## Current behavior

Start live server and see the console.log printing the message once. Change the message, save and the console.log will print twice.

## Expected behavior

print only once

## Environment

simple html + js.

Browser:

- [X] Chrome (desktop) version 68.0.3440.75
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: 5.1.1
- Platform:  Windows 10 Pro (1803, build 17134.191)
- Visual Studio Code: 1.25.1
```

## Others

**Comment by ritwickdey** [comment]
Strange! Can you please share a screenshot of your HTML & other extensions that you're using `code --list-extensions`?

**Comment by camilopedroso** [comment]
Hey! thanks for the quick response!

I'll do better: please watch it: [https://www.youtube.com/watch?v=YXUsDgWtcAE](https://www.youtube.com/watch?v=YXUsDgWtcAE)

![code list extensions](https://user-images.githubusercontent.com/872336/43366010-5a2268aa-930c-11e8-926d-05479a9022da.png)

**Comment by rjoydip** [comment]
After watching the video guessing this is an issue. @ritwickdey what you think?

**Comment by ritwickdey** [comment]
yep, I agree! but I tried my friends' laptop also, I don't see this type of issue.... 

and I'm now little bit busy... the issue is really strange...

**Comment by ritwickdey** [comment]
Hi @camilopedroso, can you please try following?

1. Open Vscode with required files & start live server. 
2. Now Open the javascript file in Notepad 
3. Try to edit & save from notepad. 

Is this working expected? or the same problem?

If the same problem, can you please help me to debug (Just I'll take your 30min, we'll text chat in skype/hangout or whatever you prefer)?

**Comment by camilopedroso** [comment]
Hey guys, FYI:
I found what is causing the weird behaviour and I want to share with you!
When I open a folder in a OneDrive synced folder, for example, at the desktop, the Live Server fires the refresh twice, but, when I open a regular folder, for example, c:\code, everything works fine!

Best regards,
Camilo

**Comment by rjoydip** [comment]
@camilopedroso Thanks for the sharing above issue. We will look into that too.

**Comment by camilopedroso** [comment]
Great! If you need some help to test, call me!
I love the live server extension!

**Comment by rjoydip** [comment]
Sure. Good to hear from you :smiley:
